apps/test/unit/code-studio/initApp/projectTest.js

```diff
@@ -47,6 +47,113 @@ describe('project.js', () => {
     });
   });
 
+  describe('project.getShareUrl', () => {
+    let fakeLocation;
+    const fakeProjectId = '<project-id>';
+
+    const ORIGINS = [
+      {
+        studio:'https://studio.code.org',
+        codeProjects: 'https://codeprojects.org',
+      },
+      {
+        studio:'https://test-studio.code.org',
+        codeProjects: 'https://test.codeprojects.org',
+      },
+      {
+        studio:'https://test.studio.code.org',
```

Comment:
Is this a domain we offer?

Reply:
Huh - I know there was a time in the past where we didn't care whether you wrote `test-studio` or `test.studio`, but maybe that's not true anymore?  This is easy enough to tear out, I just wanted to make sure this utility handled that case.

Reply:
Doesn't seem to work for me, and I have a feeling some of our infrastructure changes stopped them from working a while ago.  Same comment for the staging link in this file too.

Reply:
Didn't get confirmation yet, agreed, it looks like we don't support these anymore.  I've removed the tests.